applications/StructuralMechanicsApplication/python_scripts/structural_response.py

```diff
@@ -399,6 +399,12 @@ def _GetAdjointParameters(self):
                 KratosMultiphysics.Logger.PrintWarning(__name__, depr_msg)
                 solver_settings.RemoveValue("scheme_settings")
 
+            if solver_settings["model_import_settings"]["input_type"].GetString() == "use_input_model_part":
```

Comment:
this is not related to this PR is it?

Reply:
It is because it is the first time a response (stress . adjoint) is used in optimization.

Reply:
as @armingeiser said, this is in fact needed, as the adjoint model part always has to be read in as it gets modified. The option "use_input_model_part" is hence not applicable.